docs/user_guide/assessment/common_fairness_metrics.rst

```diff
@@ -8,6 +8,9 @@ Common fairness metrics
 In the sections below, we review the most common fairness metrics, as well
 as their underlying assumptions and suggestions for use. Each metric requires
 that some aspects of the predictor behavior be comparable across groups.
+Note that *common* usage does not imply *correct* usage; we discuss
```

Comment:
Important remark! Perhaps it should be even in bold or in a separate `note` environment thingie.